### PR TITLE
DEPLOY-321 / DEPLOY-399 : Enabled pv for solr,repo and removed ingress requirement

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

File: helm/alfresco-content-services/values.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ registryPullSecrets: quay-registry-secret
 repository:
   replicaCount: 1
   image:
-    repository: quay.io/alfresco/alfresco-full-content-services-community
-    tag: "latest"
+    repository: quay.io/alfresco/alfresco-dbp-repository
```
Comment (gavincornwell) [inline helm/alfresco-content-services/values.yaml]: Did you mean to include this in the commit?

File: helm/alfresco-content-services/values.yaml [hunk 2]
```diff
@@ -30,8 +30,8 @@ repository:
 share:
   replicaCount: 1
   image:
-    repository: quay.io/alfresco/alfresco-full-share
-    tag: "5.2.X"
+    repository: quay.io/alfresco/alfresco-dbp-share
```
Comment (gavincornwell) [inline helm/alfresco-content-services/values.yaml]: Did you mean to include this in the commit?

Comment (svidrascu) [reply helm/alfresco-content-services/values.yaml]: sorry about that, added them by mistake

**Comment by svidrascu** [comment]
Fixed, thank you Gavin.

**Pull request merged**